include/triggeralgs/TrackFinding/TriggerActivityMakerTrackFinding.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file TriggerActivityMakerTrackFinding.hpp
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2021.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#ifndef TRIGGERALGS_TRACKFINDING_TRIGGERACTIVITYMAKERTRACKFINDING_HPP_
+#define TRIGGERALGS_TRACKFINDING_TRIGGERACTIVITYMAKERTRACKFINDING_HPP_
+
+#include "triggeralgs/TPWindow.hpp"
+#include "triggeralgs/TriggerActivityFactory.hpp"
+#include <fstream>
+#include <vector>
+
+namespace triggeralgs {
+class TriggerActivityMakerTrackFinding : public TriggerActivityMaker
+{
+public:
+  void operator()(const TriggerPrimitive& input_tp, std::vector<TriggerActivity>& output_ta);
+  void configure(const nlohmann::json& config);
+
+private:
+  TriggerActivity construct_ta(TPWindow) const;
+
+  TPWindow check_adjacency();
+
+  TPWindow m_current_window;              // Holds collection hits only
+  uint64_t m_primitive_count = 0;
+  int check_tot() const;
+  
+  // Configurable parameters.
+  bool m_print_tp_info = false;          // Prints out some information on every TP received
+  bool m_trigger_on_adjacency = true;    // Default use of the horizontal muon triggering
```

Comment:
This variable doesn't get used. Was this intended to be an option?

Reply:
removed now